bin/hdfcoinc/pycbc_page_sensitivity

```diff
@@ -261,10 +261,10 @@ pycbc.results.save_fig_with_metadata(fig, args.output_file,
      title="Sensitive %s vs %s: binned by %s using %s method"
             % (args.dist_type.title(), args.sig_type.upper(), 
                args.bin_type, args.integration_method),
-     caption="Sensitive %s as a function of significance:"
-             "Darker lines represent the significance without including "
-             "injections in their own background, while lighter lines "
-             "include each injection individually in the background "
+     caption="Sensitive %s as a function of Significance:"
+             "Darker lines represent the significance including "
+             "injections in the background estimation, while lighter lines "
+             "exclude each injection individually in the background "
```

Comment:
To be pedantic here: You cannot "exclude each injection individually" in the background. The injections are not in the background, and they must be *added* individually for the darker lines. Maybe easier to invert the caption by just flipping "darker" with "lighter":

> Lighter lines represent the significance without including injections in their own background, while darker lines
include each injection individually in the background